coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMap.scala

```diff
@@ -53,7 +53,18 @@ trait BasePostgresDatasetMapReader[CT] extends `-impl`.BaseDatasetMapReader[CT]
       }
     }
 
-  def latestQuery = "SELECT system_id, copy_number, lifecycle_stage :: TEXT, data_version, last_modified FROM copy_map WHERE dataset_system_id = ? AND lifecycle_stage <> 'Discarded' ORDER BY copy_number DESC LIMIT 1"
+  def latestQuery =
+    """SELECT
+      |  system_id, copy_number, lifecycle_stage :: TEXT, data_version, last_modified, table_modifier
+      |FROM
+      |  copy_map
+      |  LEFT OUTER JOIN copy_map_table_modifiers ON copy_map.system_id = copy_map_table_modifiers.copy_system_id
+      |WHERE
+      |  dataset_system_id = ?
+      |  AND lifecycle_stage <> 'Discarded'
+      |ORDER BY
+      |  copy_number DESC
```

Comment:
We just added order by data_version desc in case there are more than one copies.